KingfisherWebP/Classes/CGImage+WebP.m

```diff
@@ -7,52 +7,169 @@
 //
 
 #import "CGImage+WebP.h"
+
+#import <Accelerate/Accelerate.h>
+
 #import "webp/decode.h"
 #import "webp/encode.h"
+#import "webp/demux.h"
 
-static void ReleaseWebPConfig(void *info, const void *data, size_t size)
-{
-    WebPDecoderConfig* config = (WebPDecoderConfig*)info;
-    WebPFreeDecBuffer(&config->output);
-    free(config);
+#pragma mark - Perfomance Settings
+
+const BOOL useThreads = true; //speed up 23%
+const BOOL bypassFiltering = false; //speed up 11%, cause some banding
+const BOOL noFancyUpsampling = false; //speed down 16%, lose some details
```

Comment:
Agreed, but re-thinking about it, took a different approach, and put it in the Swift code as is should depend on the method calling it and maybe in the future we need to allow more customization